### ISSUE TITLE
Accessibility (WCAG level AA):  Stylesheet deliberately turns off focus indicator on links

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1294]
To repro:
1.  Navigate to http://nuget.org
2.  Using the keyboard only (i.e. not eh mouse), tab to one of the links on the page.

Result:  The status bar in your browser (if enabled) will indicate that you've tabbed to a link, but there will be no focus indicator shown onscreen to indicate which link currently has focus.
Expected:  This should be visible.  The presence of a focus indicator is a AA requirement in WCAG.
Note:  This is controlled by CSS.  In Site.css, there is a setting on all a elements that sets "outline: none", which is what triggers this behavior.  Getting a focus indicator to appear around link elements can be done by removing this rule.

**Comment by johnataylor** [comment]
We have a new gallery design.
